packages/deployer/README.md

```diff
@@ -1,132 +1,61 @@
+**The recommended way to deploy Airnode is by using the [Deployer Docker](../../docker/README.md) image. This package simply implements the CLI used by that image and is not meant to be used directly by the end user.**
 
-## Temporary instructions
-
-At the root
-```sh
-yarn run build-all
-```
-In this package
-```sh
-yarn run webpack
-yarn run webpack:copy-config
-yarn run command:deploy:nonstop
-yarn run command:remove-with-receipt
-```
-
-The information below is outdated.
+## Prerequisites
+* Install [Terraform v0.15.x](https://www.terraform.io/downloads.html) and make sure that the `terraform` binary is available in your `PATH`
+* Make sure your AWS credentials are stored in the [configuration file](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-where) or exported as [environment variables](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html#envvars-set)
 
 ## Setup
-
-- Download the [Terraform v0.13.* binary](https://www.terraform.io/downloads.html) and move it to your `PATH`
-- Get an *Access Key ID* and *Secret Access Key* from AWS and insert into `.env` in the package directory (refer to `.env.example`)
-- Install Serverless Framework globally
-
-```sh
-npm install -g serverless
+* Build all the Airnode packages
+```bash
+# Run from the root
+yarn build
+```
+* Make sure `config.json` and `secrets.env` are available in the `config` directory
+```bash
+# From this package (/packages/deployer)
+cp config/config.json.example config/config.json
+cp config/secrets.env.example config/secrets.env
+# Edit both `config.json` and `secrets.env` to reflect your configuration
 ```
 
-## Basics
-
-Airnode is deployed in two steps:
-
-1. Deploy the mnemonic at AWS SSM at the `region` that the serverless functions will be deployed at using Terraform
-
-1. Deploy the serverless functions at the target `region` using Serverless Framework
-
-The mnemonic deployment needs to be performed only when deploying for the first time on a region.
-For the following redeployments, mnemonic deployment must not be repeated.
-
-## Regular user flow
-
-1. Use the `deploy-first-time` for your first deployment.
-Note down the displayed mnemonic.
-Keep the outputted receipt file, or send it to API3 if you are an API3 partner.
-
-1. Fund the master wallet address with the displayed amount to set your Airnode parameters.
-This will be done for you if you are an API3 partner.
-
-1. Use the `redeploy` command when you need to update your node.
-Note that your `airnodeIdShort` from your receipt file must be inserted into the `nodeSettings.airnodeIdShort` field of your `config.json` file.
-This will be done for you if you are an API3 partner.
+## Common user flow
+1. Use the `deploy` command for your first deployment. Write down the displayed mnemonic. Keep the outputted receipt file. Fund the master wallet address with the displayed amount to set your Airnode parameters.
```

Comment:
I think we might want to emphasis the writing down of the mnemonic and safe storage of the receipt file. They are pretty easy to miss here